translations/mod-fqm-manager/en.json

```diff
@@ -200,6 +200,7 @@
   "entityType.drv_fund_type_details.version": "Version",
   "entityType.simple_group_details": "Group",
   "entityType.simple_group_details.created_by_user_id": "Created by user UUID",
+  "entityType.simple_group_details.groups_jsonb": "JSONB",
```

Comment:
do we need translations if these are never exposed to the user?

Reply:
right now there is no way to hide it from user until we implement that hidden field story